frontend/src/components/activePage/TableTemplate/TableTemplate.tsx

```diff
@@ -38,9 +40,7 @@ const TableTemplate: FC<ITableTemplateProps> = ({ ...props }) => {
   } = props;
   const { tenantId } = user;
   const [expandedId, setExpandedId] = useState(
-    window.sessionStorage
-      .getItem(`${AVID_SSKey}Template-${templates[0].workspaceId}`)
-      ?.split(',') ?? []
+    expandedT.get(templates[0].workspaceId).split(',') ?? []
```

Comment:
```suggestion
    expandedT.get(templates[0].workspaceId).split(',')
```
Should not be needed